### PR TITLE
afterpay name renamed to Afterpayofficial

### PR DESCRIPTION
**Pull request by rmarinleal**

### CODE REVIEW

File: afterpayofficial.php [hunk 1]
```diff
@@ -109,7 +114,7 @@ class Afterpay extends PaymentModule
      */
     public function __construct()
     {
-        $this->name = 'afterpay';
+        $this->name = 'afterpayofficial';
```
Comment (pgarcess) [inline afterpayofficial.php]: use the CONST

File: controllers/front/notify.php [hunk 2]
```diff
@@ -157,7 +157,10 @@ public function postProcess()
             $this->getAfterpayOrder();
             $this->validateAmount();
             $this->checkMerchantOrderStatus();
+
         } catch (\Exception $exception) {
+            var_dump($exception->getMessage());
```
Comment (pgarcess) [inline controllers/front/notify.php]: die

**Review by pgarcess — CHANGES_REQUESTED** [review]
Comments

File: controllers/front/notify.php (hunk 2 above)
Comment (rmarinleal) [reply controllers/front/notify.php]: deleted the trace

File: afterpayofficial.php (hunk 1 above)
Comment (rmarinleal) [reply afterpayofficial.php]: done

**Pull request merged**